ci_support/run_docker_build.sh

```diff
@@ -41,10 +58,16 @@ conda clean --lock
 conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
-# Embarking on 1 case(s).
-    set -x
-    export CONDA_PERL=5.20.3.1
-    set +x
-    conda build /recipe_root --quiet || exit 1
-    upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
+conda build /recipe_root --quiet
+upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
+
+touch /feedstock_root/build_artefacts/conda-forge-build-done
 EOF
+
+echo "result from docker run: $?"
+
+# double-check that the build got to the end
+# see https://github.com/conda-forge/conda-smithy/pull/337
+# for a possible fix
+set -x
+test -f "$FEEDSTOCK_ROOT/build_artefacts/conda-forge-build-done" || exit 1
```

Comment:
I don't think that you need this last line here

Reply:
That's autogenerated by `conda-smithy`. Please see PR ( https://github.com/conda-forge/conda-smithy/pull/337 ) for details.

Reply:
Oh! Sorry you are right, I was looking at the wrong file

Reply:
No worries. It is a little strange, but there is a motivating use case for it. Maybe we can tweak the comment above to be more clear?